### PR TITLE
Add typings

### PR DESCRIPTION
**Pull request by HarelM**
This PR adds typings to this library.
It might not be complete as I have reverse engineered the code...

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+declare type IsSuppotredOptions = {
```
Comment (mourner) [inline index.d.ts]: Nit: typo here

**Review by mourner — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (HarelM) [reply index.d.ts]: My bad... fixed...

**Review by mourner — APPROVED** [review]

**Comment by HarelM** [comment]
Thanks for taking the time to review and merge this! please let me know when a new version is available.

**Comment by mourner** [comment]
Released as v2.0.1.

**Comment by HarelM** [comment]
Thanks!! [vector-tile-js](https://github.com/mapbox/vector-tile-js) is the last one :-)

**Pull request merged**